server/serverMethods.js

```diff
@@ -639,23 +636,20 @@ function ServerMethods(aLogLevel, aModules) {
     return parsed;
   }
 
-  function lockRoom(aReq, aRes) {
+  async function lockRoom(aReq, aRes) {
     var roomName = aReq.params.roomName.toLowerCase();
-    serverPersistence
-      // eslint-disable-next-line consistent-return
-      .getKey(redisRoomPrefix + roomName).then((room) => {
-        if (!room) return aRes.status(404).send(null);
+    let room = await serverPersistence.getKey(redisRoomPrefix + roomName);
+    if (!room) return aRes.status(404).send(null);
 
```

Comment:
i think we can improve this by sending error response instead of null

Reply:
I agree, but lets do it in a separate PR to keep concerns separate

Reply:
https://tokbox.atlassian.net/browse/ECO-5793